tool_commands/raster_checker/test_raster_checker.py

```diff
@@ -88,26 +92,25 @@ def test_entrees_metadata(self):
             (2, "v2_global_settings", "frict_coef_file", "rasters/test2.tif"),
             (2, "v2_interflow", "porosity_file", "rasters/test1.tif"),
         ]
-        self.assertEqual(len(entrees), len(entrees_meta_expect))
         self.assertEqual(sorted(entrees), sorted(entrees_meta_expect))
 
 
 class TestRasterChecker(unittest.TestCase):
     """Test the QGIS Environment"""
 
-    def setUp(self):
-        here = os.path.split(os.path.abspath(__file__))[0]
-        sqlite_filename = "small_2019_01_2entree_4tiff.sqlite"
-        self.test_sqlite_path = os.path.join(here, "data", sqlite_filename)
-        self.test_sqlite_dir = os.path.split(self.test_sqlite_path)[0]
-        db_type = "spatialite"
-        db_set = {"db_path": self.test_sqlite_path}
-        self.db = ThreediDatabase(db_set, db_type)
-        self.checker = RasterChecker(self.db)
-        self.entrees_expect = {
-            1: ("rasters/test1.tif", "rasters/test2.tif"),
-            2: ("rasters/test3.tif", "rasters/test2.tif", "rasters/test12.tif"),
-        }
+    def setUp(self,):
+        self.test_sqlite_dir = TEST_DATA_DIR
+        db = mock.MagicMock()
+        self.checker = RasterChecker(db)
+        # we do not need a sqlite, but we need a sqlite_dir (3di rasters are expected
+        # in a subfolder of sqlite_dir. Usually checker.sqlite_dir is set with db,
+        # which is mocked in TestRasterChecker. We set it now
+        self.checker.sqlite_dir = self.test_sqlite_dir
```

Comment:
Op regel 102 wijs je de TEST_DATA_DIR al ergens aan toe en hier effectief ook weer.
Kan je niet overal gewoon TEST_DATA_DIR gebruiken?

Reply:
ja kan zeker, zat ik gisteren ook aan te denken. Reden dat ik het niet deed was dat het duidelijk moet zijn dat de locatie van rasters in subfolder van sqlite_dir worden verwacht. 
Enfin, ik heb jouw suggestie alsnog doorgevoerd